README.adoc

```diff
@@ -8,11 +8,23 @@ This program is an onboard media control for the University of Calgary's Solar C
 :imagesdir: assets/images
 image::plan.jpg[Plan]
 
+== Acquiring Music From AWS
+
+In order to download all songs from AWS, the AWS command line tool is used to find all the song names on the server. Run the following lines in the linux terminal to aquire the commands. 
```

Comment:
Reword to 
`In order to download all songs from AWS, install the AWS command line tool:`